src/common/transformations/tests/utils/convert_precision.cpp

```diff
@@ -323,6 +324,37 @@ TEST(TransformationTests, ConvertPrecision_TopK) {
     ASSERT_FALSE(has_type<ngraph::element::Type_t::i64>(f));
 }
 
+TEST(TransformationTests, ConvertPrecision_Unique10) {
+    std::shared_ptr<ov::Model> model(nullptr);
+    {
+        auto input = std::make_shared<op::Parameter>(element::f16, Shape{15, 20, 3});
+        auto unique = std::make_shared<opset10::Unique>(input);
+
+        model = std::make_shared<ov::Model>(unique->outputs(), ParameterVector{input});
+
+        pass::Manager manager;
+
+        static const precisions_array precisions = {
```

Comment:
Missing `auto`?

Reply:
Nope, `precisions_array` is the type.
https://github.com/openvinotoolkit/openvino/blob/823ea7c68a04bbbac74cc701375ec1f2c08aff7e/src/common/transformations/include/transformations/convert_precision.hpp#L74